sgt-back-end/index.js

```diff
@@ -0,0 +1,142 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgress://dev:lfz@localhost/studentGradeTable'
+});
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grade = result.rows;
+      res.status(200).json(grade);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({ error: 'An unexpected error occurred.' });
+    });
+});
+
+app.post('/api/grades', (req, res, next) => {
+  const newName = req.body.name;
+  const newCourse = req.body.course;
+  const newScore = req.body.score;
+
+  if (newScore > 100 || newScore < 1) {
+    res.status(400).json({ error: 'Score invalid. Please input number from 1-100.' });
+    return;
+  }
+
+  const sql = `
+    insert into "grades" ("name", "course", "score")
+    values ($1, $2, $3)
+    returning *
+  `;
+  const params = [newName, newCourse, newScore];
+
+  db.query(sql, params)
+    .then(result => {
+      res.status(201).json({
+        name: newName,
+        course: newCourse,
+        score: newScore
+      });
```

Comment:
The client should also receive the `gradeId` and `createdAt` too. Your `returning` clause ensures that `result.rows[0]` will be what they're expecting.